### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by erlangp**
little clean up

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ buildscript {
 }
 
 dependencies {
-  annotationProcessor 'com.uber:rave-compiler:2.0.0'
-  compile 'com.uber:rave:2.0.0'
+    annotationProcessor 'com.uber:rave-compiler:2.0.0'
```
Comment (hzsweers) [inline README.md]: We use 2 space indents, please revert this

File: README.md [hunk 2]
```diff
@@ -107,6 +108,7 @@ public class SimpleModel {
 ### Step 3: Start Validating
 
 ```java
+
```
Comment (hzsweers) [inline README.md]: This isn't necessary, remove it. Same with the one added below

File: README.md (hunk 1 above)
Comment (erlangp) [reply README.md]: But Android Studio - Auto Format give 4 indents

Comment (hzsweers) [reply README.md]: the default maybe, but each project can have their own. Our style is 2

File: README.md (hunk 2 above)
Comment (erlangp) [reply README.md]: ok

File: README.md (hunk 1 above)
Comment (erlangp) [reply README.md]: ok

**Pull request merged**